README.md

```diff
@@ -754,6 +755,9 @@ Returns `True` if [xobni](https://www.xobni.com) chrome extension is installed `
 
 Returns `True` if [Signal](https://trysignal.com) chrome extension is installed `False` otherwise
 
+#### gmail.check.is_buttensLabel_text()
+
+Returns `True` if Button labels setting is set to Text and `False` if it is set to Icons
 
```

Comment:
You've added the new function to the documentation which is good.

Could you also add it to the typescript definitions?